docs/pages/Configuration.md

```diff
@@ -330,3 +331,16 @@ Version 1.0 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli)
 Adds support for the `theme-library` type.
 
 Version 1.1 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli) v1.13.0 and above.
+
+### Specification Version 2.0
+
+- Adds and enforces schema validation of ui5.yaml
+- The namespace for application and library projects is now mandatory.
```

Comment:
This is actually Specification Version independent. Namespace is required for all projects starting with UI5 Tooling 2.0.

So this should be removed here.